documentation/assemblies/tracing/assembly-distributed-tracing.adoc

```diff
@@ -0,0 +1,61 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-distributed-tracing-{context}']
+= Distributed tracing
+
+Distributed tracing allows you to track the progress of transactions between applications in a distributed system. In a microservices architecture, tracing tracks the progress of transactions between services. Trace data is useful for monitoring application performance and investigating issues with target systems and end-user applications.
+
+In Strimzi, tracing facilitates the end-to-end tracking of messages: from source systems to Kafka, and then from Kafka to target systems and applications. It complements the metrics that are available to view in link:{BookURLDeploying}#assembly-metrics-setup-{context}[Grafana dashboards], as well as the component loggers. 
+
+[discrete]
+== How Strimzi supports tracing
+
+Support for tracing is built in to the following components:
+
+* Kafka Connect (including Kafka Connect with Source2Image support)
+* MirrorMaker
```

Comment:
I guess we should add MirrorMaker2 as well, @scholzj is working on PR fixing tracing on it.

Reply:
Good suggestion, thanks. I've added MirrorMaker 2.0 in a few different places.